components/org.wso2.carbon.deployment.engine/pom.xml

```diff
@@ -75,6 +75,16 @@
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.utils</groupId>
+            <artifactId>carbon-utils</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.config</groupId>
+            <artifactId>org.wso2.carbon.config</artifactId>
+            <optional>true</optional>
```

Comment:
If we have used import in the java class of these dependencies, then we need to remove the optional tag.